README.md

```diff
@@ -1,20 +1,22 @@
 # Serverless Webpack
 
-[![serverless](http://public.serverless.com/badges/v3.svg)](http://www.serverless.com)
-[![CircleCI](https://circleci.com/gh/elastic-coders/serverless-webpack.svg?style=shield)](https://circleci.com/gh/elastic-coders/serverless-webpack)
+[![Serverless][ico-serverless]][link-serverless]
+[![CircleCI][ico-circleci]][link-circleci]
+[![NPM][ico-npm]][link-npm]
```

Comment:
The icons are already SVG unless I'm mistaken... if you check the bottom of `README.md`:

```
[ico-serverless]: http://public.serverless.com/badges/v3.svg
[ico-circleci]: https://img.shields.io/circleci/project/github/elastic-coders/serverless-webpack.svg
[ico-npm]: https://img.shields.io/npm/v/serverless-webpack.svg
[ico-contributors]: https://img.shields.io/github/contributors/elastic-coders/serverless-webpack.svg
```